app/marketing/models.py

```diff
@@ -26,6 +26,21 @@
 from economy.models import SuperModel
 
 
+class Alumni(SuperModel):
+
+    username = models.CharField(max_length=255, db_index=True)
```

Comment:
Should we consider relating this to `Profile` since anyone that's alum should have a gitcoin profile?  Then we could eliminate the duplicate data of `username` and functionality like `avatar_url`.

Reply:
yea probably

Reply:
Plus once we start using `Groups` / `Organizations` we could simply add (or propose an addition to the user) to `Alumni` based on addition to a specific org following following their relation to any given org existing on the platform.

Reply:
5dd8c6fa825f12a189decde2cac9de27b23ce2da